.github/workflows/ci.yml

```diff
@@ -69,3 +69,33 @@ jobs:
 
     - name: Run PHP Code Sniffer
       run: composer ci:php:sniff
+
+  unit-tests:
+    name: Unit tests
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version:
+          - 7.2
+          - 7.3
+          - 7.4
```

Comment:
We'll also need to run the tests with TYPO3 9 and 10.

Reply:
This is currently not possible since the dev-dependencies do not work with TYPO3 v10.